addons/dev-helpers/setup-go-env.sh

```diff
@@ -14,6 +14,7 @@ else
   echo 'export PATH=$PATH:/usr/local/go/bin' >> ~/.bashrc
   echo 'export GOPATH=~/gospace' >> ~/.bashrc
   echo 'export PATH=~/gospace/bin:$PATH' >> ~/.bashrc
+  GOPATH=~/gospace
```

Comment:
What you suggested will also do the trick.
Adjust your PR and we're good to merge this